packages/babel-plugin-transform-runtime/test/fixtures/absoluteRuntime/true-corejs3-stable/options.json

```
@@ -0,0 +1,6 @@
+{
+  "presets": ["env"],
+  "plugins": [
+    ["transform-runtime", { "absoluteRuntime": true, "corejs": { "version": 3, "proposals": false } }]
```

nit:
```suggestion
    ["transform-runtime", { "absoluteRuntime": true, "corejs": { "version": 3, "proposals": false } }],
    ["external-helpers", { "helperVersion": "7.100.0" }]
```
will remove helper contents in the `output.js`, so we don't have to update many fixtures when helper codes are changed.